examples/Upbit/Spot.jl

```diff
@@ -0,0 +1,22 @@
+# Upbit/Spot
+# https://docs.upbit.com
+
+using Dates
+using CryptoAPIs
+using CryptoAPIs.Upbit
+
+CryptoAPIs.Upbit.Spot.day_candle(;
+    market = "KRW-BTC",
+    convertingPriceUnit = "KRW",
+    count = 1,
+    to = DateTime("2023-01-01T00:00:00")
+)
+
+CryptoAPIs.Upbit.Spot.market_list(; isDetails = true)
+
+CryptoAPIs.Upbit.Spot.order_book(; markets = "KRW-BTC")
+
+CryptoAPIs.Upbit.Spot.status_wallet()
+
+CryptoAPIs.Upbit.Spot.ticker(; markets = "KRW-BTC")
+CryptoAPIs.Upbit.Spot.ticker(; markets = ["KRW-BTC", "BTC-ETH"])
```

Comment:
error
```julia
error = "UpbitAPIError{404}(name = "404", msg = "Code not found")"
```

Reply:
It happens because of this
```julia
function Serde.SerQuery.ser_type(::Type{TickerQuery}, x::Vector{String})::String
    return "[\"" * join(x, "\",\"") * "\"]"
end
```

Reply:
Would it be better to rewrite it like this?
`
function Serde.SerQuery.ser_type(::Type{TickerQuery}, x::Vector{String})::String
    return join(x, "\",\"")
end
`

Reply:
it is worth deleting altogether `Serde.SerQuery.ser_type(::Type{TickerQuery}, x::Vector{String})::String`, because field `markets` can accept when entering the `Vector{String}`